contracts/governance/scripts/AddOperator.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.11;
+
+import "../../interfaces/INestedFactory.sol";
+import "../../interfaces/IOperatorResolver.sol";
+import "../../abstracts/MixinOperatorResolver.sol";
+
+contract AddOperator {
```

Comment:
Could be great to also have the RemoveOperator ready to prevent making mistakes if we ever need to do it urgently.